pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -634,9 +629,17 @@ def _auto_select_batch_size(self):
         batch_size = 1
         train_dl_source = self.lightning_module.trainer._data_connector._train_dataloader_source
         if train_dl_source.is_defined():
-            train_dataloader = train_dl_source.dataloader()
-            if hasattr(train_dataloader, "batch_sampler"):
-                batch_size = train_dataloader.batch_sampler.batch_size
+            try:
+                train_dataloader = train_dl_source.dataloader()
+                if hasattr(train_dataloader, "batch_sampler"):
+                    batch_size = train_dataloader.batch_sampler.batch_size
+            except Exception:
+                if deepspeed.utils.logging.logger.level < logging.WARN:
```

Comment:
shouldn't this be <=?

Reply:
Don't think so, with `logging.WARN` the logging that uses the batch size are supressed